mailbox/api/src/main/java/org/apache/james/mailbox/MailboxManager.java

```diff
@@ -328,4 +330,8 @@
      */
     List<MailboxPath> list(MailboxSession session) throws MailboxException;
 
-}
+    List<MailboxAnnotation> getAnnotations(MailboxPath mailboxPath, MailboxSession session) throws MailboxException;
```

Comment:
you should write javadoc for methods you add to this interface, it's something public that needs precise description of the expected behavior